cmd/mdatagen/loader.go

```diff
@@ -45,15 +50,13 @@ type metric struct {
 
 	// ExtendedDocumentation of the metric. If specified, this will
 	// be appended to the description used in generated documentation.
-	ExtendedDocumentation string `yaml:"extended_documentation"`
+	ExtendedDocumentation string `mapstructure:"extended_documentation"`
 
 	// Unit of the metric.
-	Unit string `yaml:"unit"`
+	Unit string
```

Comment:
Why removing the `tag`?

Reply:
This must be parsed as `"unit"` key by default. Looks like we always specify it in other places. I'll bring it back